elasticsearch/manifests/configmap.yaml

```diff
@@ -31,7 +31,8 @@ data:
   post-start-hook.sh: |-
     #!/bin/bash
     set -e
-    apk --no-cache add jq
+    yum install -y epel-release
```

Comment:
Is jq installed? I really hate this code here.

Reply:
`jq` isn't already installed in the elasticsearch container:

![1420](https://user-images.githubusercontent.com/18330710/50578003-6df72d00-0de8-11e9-8093-bd3e49d66d1b.jpg)

I took out the `jq` bits in that shut down script replaced with them with some inline python.